pkg/authorization/admission/restrictusers/restrictusers_test.go

```diff
@@ -569,6 +572,40 @@ func TestAdmission(t *testing.T) {
 	stopCh := make(chan struct{})
 	defer close(stopCh)
 
+	// for each testCase that involves an authorizationapi rolebinding also
```

Comment:
I would like one command line based test for each type (i.e. I want this verified with a real server).